pkg/proxy/ipvs/proxier.go

```diff
@@ -402,6 +403,11 @@ func NewProxier(ipFamily v1.IPFamily,
 		}
 	}
 
+	if initOnly {
```

Comment:
Would it make sense to have a comment here (and in `iptables/proxies.go`) that anything privileged should come before this point?

Reply:
Well it might, but I leave it for now just to get it done :smiley: